instagram/src/App.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from "react";
+import dummyData from "./dummy-data";
+import SearchBar from "./components/SearchBar/SearchBar";
+import PostContainer from "./components/PostContainer/PostContainer";
+
+import "./css/main.css";
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = { data: dummyData };
+  }
+
+  changeHandler = id => {
+    console.log(id);
+  };
+
+  render() {
+    return (
+      <div className="container">
+        <header className="header">
+          <SearchBar changeHandler={this.changeHandler} />
+        </header>
```

Comment:
consider your rendered DOM structure for this sections: you only have one child element of SearchBar inside this header, and inside SearchBar you also have only one child div wrapping everything. I'd say you could take this header wrapper out and add all the relevant styles to the returned div instead